src/sync/sync_config.hpp

```diff
@@ -100,6 +104,7 @@ struct SyncConfig {
     SyncSessionStopPolicy stop_policy;
     std::function<SyncBindSessionHandler> bind_session_handler;
     std::function<SyncSessionErrorHandler> error_handler;
+    std::shared_ptr<ChangesetTransformer> transformer;
```

Comment:
I can see that you are setting `sync::SyncHistory::Config::changeset_cooker` here https://github.com/realm/realm-object-store/pull/363/files#diff-55a15b0c9125ec5f7989470e6eb8b217R182. If I am right, and I'm pretty sure I am, you should be able to remove that line and have everything working the same as before.